Fix incorrect labels definition for systemtest

lahsivjar opened the pull request:
`labels` should be inside `metadata` rather than at the same level as `metadata`:

```
{
    "metadata": {
        "service": {...},
        "labels": {...},
        ...
    }
}
```

apmmachine commented:
## :green_heart: Build Succeeded


<!-- BUILD BADGES-->
> _the below badges are clickable and redirect to their specific view in the CI or DOCS_
[![Pipeline View](https://img.shields.io/badge/pipeline-pipeline%20-green)](https://apm-ci.elastic.co/blue/organizations/jenkins/apm-server%2Fapm-server-mbp%2FPR-10717/detail/PR-10717/1//pipeline) [![Test View](https://img.shields.io/badge/test-test-green)](https://apm-ci.elastic.co/blue/organizations/jenkins/apm-server%2Fapm-server-mbp%2FPR-10717/detail/PR-10717/1//tests) [![Changes](https://img.shields.io/badge/changes-changes-green)](https://apm-ci.elastic.co/blue/organizations/jenkins/apm-server%2Fapm-server-mbp%2FPR-10717/detail/PR-10717/1//changes) [![Artifacts](https://img.shields.io/badge/artifacts-artifacts-yellow)](https://apm-ci.elastic.co/blue/organizations/jenkins/apm-server%2Fapm-server-mbp%2FPR-10717/detail/PR-10717/1//artifacts) [![preview](https://img.shields.io/badge/docs-preview-yellowgreen)](http://apm-server_10717.docs-preview.app.elstc.co/diff)  [![preview](https://img.shields.io/badge/elastic-observability-blue)](https://ci-stats.elastic.co/app/apm/services/apm-ci/transactions/view?rangeFrom=2023-04-28T06:00:11.858Z&rangeTo=2023-04-28T06:20:11.858Z&transactionName=BUILD+apm-server%2Fapm-server-mbp%2FPR-%7Bnumber%7D&transactionType=job&latencyAggregationType=avg&traceId=c7ff2dd41eb635ed67170975f22b5d97&transactionId=2f982131e8ea56ce)

<!-- BUILD SUMMARY-->
<details><summary>Expand to view the summary</summary>
<p>

#### Build stats




* Start Time: 2023-04-28T06:10:11.858+0000


* Duration: 7 min 15 sec





</p>
</details>

<!-- TEST RESULTS IF ANY-->


<!-- STEPS ERRORS IF ANY -->














## :robot: GitHub comments

<details><summary>Expand to view the GitHub comments</summary>
<p>

To re-run your PR in the CI, just comment with:



- `/test` : Re-trigger the build.


- `/package` : Generate and publish the docker images.


- `/test windows` : Build & tests on Windows.


- `run` `elasticsearch-ci/docs` : Re-trigger the docs validation. (use unformatted text in the comment!)




</p>
</details>

<!--COMMENT_GENERATED_WITH_ID_comment.id-->

apmmachine commented:
### :books: Go benchmark report

Diff with the `main` branch

```
goos: linux
goarch: amd64
pkg: github.com/elastic/apm-server/internal/agentcfg
cpu: 12th Gen Intel(R) Core(TM) i5-12500
                                  │ build/main/bench.out │             bench.out              │
                                  │        sec/op        │   sec/op     vs base               │
FetchAndAdd/FetchFromCache-12                46.09n ± 0%   41.12n ± 0%  -10.79% (p=0.002 n=6)
FetchAndAdd/FetchAndAddToCache-12           103.35n ± 4%   91.25n ± 3%  -11.71% (p=0.002 n=6)
geomean                                      69.02n        61.25n       -11.25%

                                  │ build/main/bench.out │             bench.out              │
                                  │         B/op         │    B/op     vs base                │
geomean                                                ²               +0.00%               ²
¹ all samples are equal
² summaries must be >0 to compute geomean

                                  │ build/main/bench.out │             bench.out              │
                                  │      allocs/op       │ allocs/op   vs base                │
geomean                                                ²               +0.00%               ²
¹ all samples are equal
² summaries must be >0 to compute geomean

pkg: github.com/elastic/apm-server/internal/beater/request
                                             │ build/main/bench.out │              bench.out              │
                                             │        sec/op        │    sec/op     vs base               │
ContextResetContentEncoding/empty-12                   134.8n ±  1%   118.5n ±  1%  -12.10% (p=0.002 n=6)
ContextResetContentEncoding/uncompressed-12            161.2n ±  1%   142.3n ±  1%  -11.69% (p=0.002 n=6)
geomean                                                907.9n         938.1n         +3.33%

                                             │ build/main/bench.out │              bench.out               │
                                             │         B/op         │     B/op      vs base                │
geomean                                                           ²                 +0.00%               ²
¹ all samples are equal
² summaries must be >0 to compute geomean

                                             │ build/main/bench.out │             bench.out              │
                                             │      allocs/op       │ allocs/op   vs base                │
geomean                                                           ²               +0.00%               ²
¹ all samples are equal
² summaries must be >0 to compute geomean

pkg: github.com/elastic/apm-server/internal/publish
             │ build/main/bench.out │          bench.out          │
             │        sec/op        │   sec/op    vs base         │

             │ build/main/bench.out │           bench.out            │
             │         B/op         │     B/op       vs base         │

             │ build/main/bench.out │           bench.out           │
             │      allocs/op       │  allocs/op    vs base         │

pkg: github.com/elastic/apm-server/x-pack/apm-server/aggregation/spanmetrics
                 │ build/main/bench.out │          bench.out           │
                 │        sec/op        │   sec/op     vs base         │

                 │ build/main/bench.out │            bench.out            │
                 │         B/op         │     B/op      vs base           │
¹ all samples are equal

                 │ build/main/bench.out │           bench.out           │
                 │      allocs/op       │ allocs/op   vs base           │
¹ all samples are equal

pkg: github.com/elastic/apm-server/x-pack/apm-server/aggregation/txmetrics
                        │ build/main/bench.out │          bench.out           │
                        │        sec/op        │   sec/op     vs base         │

                        │ build/main/bench.out │           bench.out           │
                        │         B/op         │    B/op     vs base           │
¹ all samples are equal

                        │ build/main/bench.out │           bench.out           │
                        │      allocs/op       │ allocs/op   vs base           │
¹ all samples are equal

pkg: github.com/elastic/apm-server/x-pack/apm-server/sampling
               │ build/main/bench.out │              bench.out              │
               │        sec/op        │    sec/op     vs base               │
geomean                  660.3n         582.2n        -11.83%

               │ build/main/bench.out │              bench.out               │
               │         B/op         │     B/op      vs base                │
geomean                             ²                 -0.74%               ²
¹ all samples are equal
² summaries must be >0 to compute geomean

               │ build/main/bench.out │             bench.out              │
               │      allocs/op       │ allocs/op   vs base                │
geomean                             ²               +0.00%               ²
¹ all samples are equal
² summaries must be >0 to compute geomean

pkg: github.com/elastic/apm-server/x-pack/apm-server/sampling/eventstorage
                                            │ build/main/bench.out │              bench.out              │
                                            │        sec/op        │    sec/op     vs base               │
WriteTransaction/json_codec-12                       14.242µ ± 16%   4.091µ ±  2%  -71.27% (p=0.002 n=6)
WriteTransaction/json_codec_big_tx-12                13.847µ ± 41%   4.968µ ±  3%  -64.12% (p=0.002 n=6)
ReadEvents/json_codec/0_events-12                     355.7n ±  5%   305.4n ±  5%  -14.13% (p=0.002 n=6)
ReadEvents/json_codec/1_events-12                     11.16µ ±  6%   10.13µ ±  5%   -9.25% (p=0.009 n=6)
ReadEvents/json_codec_big_tx/0_events-12              352.2n ±  8%   310.2n ±  8%  -11.91% (p=0.004 n=6)
ReadEvents/json_codec_big_tx/399_events-12            3.771m ±  2%   3.819m ±  2%   +1.27% (p=0.026 n=6)
ReadEvents/nop_codec/0_events-12                      349.8n ±  7%   310.5n ±  9%  -11.22% (p=0.004 n=6)
ReadEvents/nop_codec_big_tx/0_events-12               342.9n ±  5%   302.4n ±  8%  -11.81% (p=0.002 n=6)
ReadEvents/nop_codec_big_tx/399_events-12             481.6µ ±  5%   437.4µ ± 11%   -9.18% (p=0.041 n=6)
IsTraceSampled/sampled-12                             75.72n ±  4%   67.59n ±  3%  -10.74% (p=0.002 n=6)
IsTraceSampled/unsampled-12                           78.31n ±  1%   70.30n ±  2%  -10.23% (p=0.002 n=6)
IsTraceSampled/unknown-12                             426.1n ±  7%   374.5n ±  4%  -12.12% (p=0.002 n=6)
geomean                                               31.43µ         29.06µ         -7.56%

                                            │ build/main/bench.out │               bench.out                │
                                            │         B/op         │      B/op       vs base                │
WriteTransaction/json_codec_big_tx-12                3.688Ki ±  0%    3.686Ki ±  0%  -0.05% (p=0.013 n=6)
ReadEvents/json_codec_big_tx/1000_events-12          4.661Mi ±  2%    4.661Mi ±  0%  -0.00% (p=0.041 n=6)
ReadEvents/nop_codec/199_events-12                   459.0Ki ±  0%    459.5Ki ±  0%  +0.11% (p=0.045 n=6)
ReadEvents/nop_codec/1000_events-12                  2.084Mi ±  0%    2.078Mi ±  0%  -0.31% (p=0.015 n=6)
ReadEvents/nop_codec_big_tx/199_events-12            458.9Ki ±  0%    459.6Ki ±  0%  +0.14% (p=0.026 n=6)
geomean                                              31.48Ki          31.42Ki        -0.20%
¹ all samples are equal

                                            │ build/main/bench.out │              bench.out              │
                                            │      allocs/op       │  allocs/op   vs base                │
geomean                                                 144.7         144.7       -0.00%
¹ all samples are equal

```

_report generated with https://pkg.go.dev/golang.org/x/perf/cmd/benchstat_
<!--COMMENT_GENERATED_WITH_ID_bench.diff-->

lahsivjar commented:
[**For reviewers**]: We were earlier using the backend endpoint for the rum events, I have updated it to use rum endpoint and updated the ES query to search for global labels for the `traces` making the test fail with timeout without this fix.

lahsivjar commented:
/test
